### PR TITLE
Level0 tasks

### PR DESCRIPTION
**Pull request by Tanakah5Star**
Realized i had made a mistake in that i did not make a pull request. Noticed my mistake as i was working on Level 1 Challenges. Hope they get Merged in time.

### CODE REVIEW

File: Task0.10.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+               resultString.push(n);
+            }
+        }
+    };
+    return resultString;
+  }
+         
+console.log(returnSubsequence("tanakah","mashoko"));*/
+
+
```
Comment (Sbonelo01) [inline Task0.10.js]: Remove this please. Your final submission should be as clean as possible.

File: Task0.8.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+function convertToHoursMinutes(number){
+    var display= " ";
+    var hours = Math.floor(number/60);//Uses Math.floor method to determin how many hours are in the number.
+    var minutes  = number % 60;//Uses the Modulus operator to get minutes remaining on the number.
+    //Making use of Ternary operators to control singulars and plurals for hours and minutes.
```
Comment (Sbonelo01) [inline Task0.8.js]: Don't comment everything. Only comment when the code is not easy to understand - not when it is self explanatory

**Review by Sbonelo01 — CHANGES_REQUESTED** [review]
Please neaten your code. Also, check how you named some of your files. e.g: Task0,10.js(Bad naming)

File: Task0.10.js (hunk 1 above)
Comment (Tanakah5Star) [reply Task0.10.js]: Hi Sbonelo. Thank you for the feedback. I noticed this and i Updated the code when i resubmitted the pull request. i think you will notice my commits named " Updated Answer to ......". I realised the commented code as well as the Bad naming that i had used and changed to Task0.1.js from Task0,1,js. I am not sure if the commits can be noticed. 
Also noted is the unnecessary comments withing some of the code and will be rectified instantly.

File: Task0.8.js (hunk 2 above)
Comment (Tanakah5Star) [reply Task0.8.js]: Thank you Sbonelo. Noted. fixing right away.

**Comment by ry-oc** [comment]
Hi there, seems like there is a bit of confusion here. For this project as it is a personal repo you need to merge your own code and move the card to review when it is merged.

**Comment by Tanakah5Star** [comment]
Thank you Ryan, merging it right away.

**Pull request merged**